docs/source/ephys_at_swc/resources.md

```diff
@@ -0,0 +1,113 @@
+# Resources
+
+This section includes articles and videos providing
+background and technical detail on extracellular
+electrophysiology data preprocessing and analysis.
+
+This section is by no means exhaustive, please
+feel free to [get in contact](community.md) to suggest
+additions to this page.
+
+## General Introduction
+
+Below are a selection of papers that give a history
+and overview of the extracellular electrophysiology landscape:
+
+[Steinmetz NA et al. (2018). Challenges and opportunities for large-scale electrophysiology with Neuropixels probes. *Current Opinion in Neurobiology*.](https://pubmed.ncbi.nlm.nih.gov/29444488/)
+
+[Buccino AP et al. (2022). Spike sorting: new trends and challenges of the era of high-density probes. *Progress in Biomedical Engineering*.](https://iopscience.iop.org/article/10.1088/2516-1091/ac6b96/meta)
+
+[Rey HG et al. (2015). Past, present and future of spike sorting techniques. *Brain Research Bulletin*.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4674014/)
+
+[Carlson D et al. (2019). Continuing progress of spike sorting in the era of big data. *Current Opinion in Neurobiology*](https://pubmed.ncbi.nlm.nih.gov/30856552/)
```

Comment:
Would it be useful to add a one sentence description of why each of these papers are useful?

Reply:
I think this is a cool idea but I would have to re-read / skim each paper in some detail to do this properly and might take quite a bit of time. As such I'll leave it for now.